packages/upgrade/test/fixtures/upgrade/ts-any.ts.output

```diff
@@ -1,8 +1,56 @@
+/**
+ * @param {String} a - some string
+ * @param {any} b - hardcoded to any just for demonstrative purposes
+ */
+/* @ts-expect-error @rehearsal TODO TS7006: Parameter 'b' implicitly has an 'any' type. */
+export function oneOfParamsHaveAny(a: string, b) {
+  console.log(a, b);
+}
+
+/**
+ * @param {String} a - some string
+ * @param {string | any} b - hardcoded to any just for demonstrative purposes
+ */
+/* @ts-expect-error @rehearsal TODO TS7006: Parameter 'b' implicitly has an 'any' type. */
+export function oneOfParamsHaveSomethingOrAny(a: string, b) {
+  console.log(a, b);
+}
+
+/**
+ * @param {String} a - A string
+ * @returns {Promise} the promise that resolves when the request has been resumed
+ */
+export function returnTypeHasGenericAny(a: string) {
+  return Promise.resolve(a);
+}
+
+/**
+ * @param {String} a - some string
+ * @param {any} b - hardcoded to any just for demonstrative purposes
+ * @returns {Promise} the promise that resolves when the request has been resumed
+ */
+/* @ts-expect-error @rehearsal TODO TS7006: Parameter 'b' implicitly has an 'any' type. */
+export function oneOfParamsHaveAnyAndReturnHasGenericAny(a: string, b) {
+  return Promise.resolve([a, b]);
+}
+
+/**
+ * @param {String} a - some string
+ * @param {any} b - hardcoded to any just for demonstrative purposes
+ * @returns {any} - just return something
+ */
+/* @ts-expect-error @rehearsal TODO TS7006: Parameter 'b' implicitly has an 'any' type. */
+export function oneOfParamsHaveAnyAndReturnIsAny(a: string, b) {
+  console.log(a);
+  return b;
+}
+
 function capitalize(str: string): string {
   return str[0].toUpperCase() + str.substring(1);
 }
 
-export function doSomething(args: string | any[], name: string) {
+/* @ts-expect-error @rehearsal TODO TS7006: Parameter 'args' implicitly has an 'any' type. */
+export function doSomething(args, name: string) {
```

Comment:
`args` type is detected as `string | any[]`, but we drop the whole suggested fix because it contains any.
We can do some work around this in the next PRs if needed.